src/components/NavBar.tsx

```diff
@@ -3,17 +3,35 @@ import H5 from "../shared/components/H5";
 import { BsFillCartFill, BsPersonFill } from "react-icons/bs";
 
 const NavBar = () => {
+  const token = localStorage.getItem("authToken");
```

Comment:
make a common util (`storage.ts`) where you can place functions to getItem, setItem, clear items from localstorage.